soh/soh/SaveManager.h

```diff
@@ -39,7 +39,10 @@ typedef struct {
 #include <functional>
 #include <vector>
 #include <filesystem>
-#include "thread-pool/BS_thread_pool.hpp"
+
+#define BS_THREAD_POOL_ENABLE_PRIORITY
+#define BS_THREAD_POOL_ENABLE_PAUSE
```

Comment:
we use these in LUS, we get errors if we include the header without them before including it in LUS with them

from https://github.com/bshoshany/thread-pool/releases/tag/v4.0.0

> by default, the thread pool is in "light mode". Optional features that may affect performance due to additional checks or more complicated algorithms must be enabled by defining suitable macros before including the library:
> `BS_THREAD_POOL_ENABLE_PAUSE` to enable pausing.
> `BS_THREAD_POOL_ENABLE_PRIORITY` to enable task priority.